.changeset/flat-games-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@infinit-xyz/core': patch
```

Comment:
Should we use a minor update for this one since it's seem to break the previous interface?